athlinks-ontology.ttl

```diff
@@ -6,542 +6,680 @@
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix schema: <http://schema.org/> .
-@base <http://adp.athlinks.com/schema> .
 
-# <http://adp.athlinks.com/schemax> rdf:type owl:Ontology; rdf:about "schema-latest-owl.ttl" .
+<http://adp.athlinks.com/schema> a owl:Ontology ;
+	owl:versionIRI <http://adp.athlinks.com/schema/> ;
+	owl:imports <http://adp.athlinks.com/schemax/> ;
+	dc:contributor "berg"^^xsd:string ;
+	rdfs:comment "This file contains Athlinks data vocabulary. The vocabulary is based on the schema.org ontology with our own extensions for concepts specific to human endurance events."@en ;
+	rdfs:label "Athlinks data vocabulary"@en ;
+	dc:contributor "astn"^^xsd:string , "nextdude"^^xsd:string , "warren"^^xsd:string .
+# 
+# 
+# #################################################################
+# #
+# #    Object Properties
+# #
+# #################################################################
+# 
+# 
+# http://adp.athlinks.com/schema/exerciseAction
+
+:exerciseAction a owl:ObjectProperty ;
+	rdfs:subPropertyOf owl:topObjectProperty ;
+	rdfs:domain :AthleteResult ;
+	rdfs:range schema:ExerciseAction ;
+	rdfs:comment "A pointer to the the exercise action that is being ranked."@en ;
+	rdfs:label "exercise action"@en .
+# 
+# http://adp.athlinks.com/schema/legacyData
+
+:legacyData a owl:ObjectProperty ;
+	rdfs:domain owl:Thing ;
+	rdfs:range :LegacyDataRole ;
+	rdfs:comment "A legacy data record belonging to another system that is related to this object."@en ;
+	rdfs:label "legacy data"@en .
+# 
+# http://adp.athlinks.com/schema/participatedIn
```

Comment:
In order to accommodate the object property link between `:AthleticEventCompetitor` and `:AthleticEvent`, we need to widen the domain of `:sportsEvent`:

```
schema:sportsEvent a owl:ObjectProperty ;
  rdfs:domain [
    rdf:type owl:Class ;
    owl:unionOf (
      schema:ExerciseAction
      :AthleticEventRole
    ) ;
  ];
  rdfs:range schema:SportsEvent .
```

Reply:
Or we could just add an `:athleticEvent` property to `:AthleticEventRole` that points to an `:AthleticEvent`. Its probably superstitious, but I feel uneasy about modifying `schema.org` properties and classes directly.